portal/exercise_diet/templates/exercise_diet/exercise-diet_portal.html

```diff
@@ -41,9 +41,9 @@ <h5>{{_("EXERCISE AND DIET")}}</h5>
 					<h6><span>{{_("EXERCISE / DIET / RECIPES")}}</span></h6>
 					</div>
 					{% if not user %}
-						<span class="toolbox-start-button" data-toggle="modal" data-target="#modal-register">{% trans %}start &rarr;{% endtrans %}</span>
+						<span class="toolbox-start-button" data-toggle="modal" data-target="#modal-register">{% trans %}start <span class="figure">&rarr;</span>{% endtrans %}</span>
 					{% else %}
-						<a href="{{url_for('exercise_diet.introduction')}}" class="exercise_diet-link community_of_wellness-link"><span class="toolbox-start-button">{% trans %}start &rarr;{% endtrans %}</span></a>
+						<a href="{{url_for('exercise_diet.introduction')}}" class="exercise_diet-link community_of_wellness-link"><span class="toolbox-start-button">{% trans %}start <span class="figure">&rarr;</span>{% endtrans %}</span></a>
```

Comment:
I don't think you need to translate that `<span> ` since it doesn't have any translatable text (eg `title`)
```suggestion
						<a href="{{url_for('exercise_diet.introduction')}}" class="exercise_diet-link community_of_wellness-link"><span class="toolbox-start-button">{% trans %}start{% endtrans %}<span class="figure">&rarr;</span></span></a>
```